scripts/magic-gen/Makefile

```diff
@@ -1,17 +1,14 @@
-GO ?= go
+PY ?= python3
 
 MAGIC_DEST ?= ../../magic/magic.go
 
 magic-gen: clean magic.h
-	$(GO) run magicgen.go magic.h $(MAGIC_DEST)
+	@$(PY) magicgen.py magic.h $(MAGIC_DEST)
 
 magic.h:
 	curl -s https://raw.githubusercontent.com/checkpoint-restore/criu/criu-dev/criu/include/magic.h -o magic.h
 
 test:
-	@echo "Running unit test"
-	$(GO) test -v
```

Comment:
Would it be possible to add unit tests for `read_magics` / `write_magics` similar to `magicgen_test.go`?